test/unit/atomic/RAJA_gtest.hpp

```diff
@@ -0,0 +1,100 @@
+/*!
```

Comment:
Shouldn't this file be in the test/include directory since it's not specific to atomic unit tests?

Reply:
Yes, I will move it there.